packages/@react-spectrum/actionbar/intl/cs-CZ.json

```diff
@@ -2,6 +2,6 @@
   "actions": "Akce",
   "actionsAvailable": "Dostupné akce.",
   "clearSelection": "Vymazat výběr",
-  "selected": "{count, plural, =0 {Nic není vybráno} other {Vybráno #}}",
+  "selected": "Vybráno: { count }",
```

Comment:
Just double checking we are ok removing the plural specifics here?

Reply:
This should be okay, as variable still present (count), not all languages work same for plural. This particular string was updated by Leandro as part of his i18n review,